README.md

```diff
@@ -15,251 +15,191 @@
-[migration-shield]: https://img.shields.io/badge/Wiki-Migrating%20from%20V3-darkgreen.svg
+[migration-shield]: https://img.shields.io/badge/Wiki-Migrating%20from%20V4-darkgreen.svg
+[GatewayIntent]: https://docs.jda.wiki/net/dv8tion/jda/api/requests/GatewayIntent.html
+[JDABuilder]: https://docs.jda.wiki/net/dv8tion/jda/api/JDABuilder.html
+[DefaultShardManagerBuilder]: https://docs.jda.wiki/net/dv8tion/jda/api/sharding/DefaultShardManagerBuilder.html
 
-<img align="right" src="https://github.com/discord-jda/JDA/blob/assets/assets/readme/logo.png?raw=true" height="200" width="200">
+<img align="right" src="https://github.com/discord-jda/JDA/blob/assets/assets/readme/logo.png?raw=true" height="150" width="150">
 
-[ ![maven-central][] ][download]
-[ ![jitpack][] ](https://jitpack.io/#discord-jda/JDA)
-[ ![jenkins-shield][] ][jenkins]
-[ ![license-shield][] ][license]
-
-[ ![discord-shield][] ][discord-invite]
-[ ![faq-shield] ][faq]
-[ ![docs-shield] ][docs]
-[ ![troubleshooting-shield] ][troubleshooting]
-[ ![migration-shield][] ][migration]
+[![maven-central][]][download]
+[![jitpack][]](https://jitpack.io/#discord-jda/JDA)
+[![jenkins-shield][]][jenkins]
+[![license-shield][]][license]
 
+[![discord-shield][]][discord-invite]
+[![faq-shield]][faq]
+[![docs-shield]][docs]
+[![troubleshooting-shield]][troubleshooting]
+[![migration-shield][]][migration]
 
 # JDA (Java Discord API)
 
-JDA strives to provide a clean and full wrapping of the Discord REST API and its WebSocket-Events for Java.
-This library is a helpful tool that provides the functionality to create a Discord bot in Java.
+This open source library is intended for implementing bots on Discord using the real-time gateway and REST API. It provides event based functionality to implement bots of any kind, allowing for effective and scalable applications.
 
-## Summary
+## 📖 Overview
 
-1. [Introduction](#creating-the-jda-object)
-2. [Sharding](#sharding-a-bot)
-3. [Entity Lifetimes](#entity-lifetimes)
-4. [Download](#download)
-5. [Documentation](#documentation)
-6. [Support](#getting-help)
-7. [Extensions And Plugins](#third-party-recommendations)
-8. [Contributing](#contributing-to-jda)
-9. [Dependencies](#dependencies)
-10. [Other Libraries](#related-projects)
+The core concepts of JDA have been developed to make building scalable apps easy:
 
-## UserBots and SelfBots
+1. Event System  
+    Providing simplified events from the gateway API, to respond to any platform events in real-time without much hassle.
+1. Rest Actions  
+    Easy to use and scalable implementation of REST API functionality, letting you choose between callbacks with combinators, futures, and blocking.
+    The library also handles rate-limits imposed by Discord automatically, while still offering ways to replace the default implementation.
+1. Customizable Cache  
+    Trading memory usage for better performance where necessary, with sane default presets to choose from and customize.
```

Comment:
Lets put a line at the bottom of this section like

"You can easily learn more by visiting `[our wiki](https://jda.wiki)` or referencing `[our Javadocs](https://docs.jda.wiki/)`